gallery_dl/extractor/aryion.py

```diff
@@ -199,6 +203,19 @@ def posts(self):
             url = "{}/g4/latest.php?name={}".format(self.root, self.user)
             return util.advance(self._pagination_next(url), self.offset)
 
+class AryionFavoriteExtractor(AryionExtractor):
+    """Extractor for a user's favorites gallery"""
+    subcategory = "favorite"
+    categorytransfer = True
```

Comment:
Is this correct here? I copied it from the gallery extractor, but I don't understand what it does.

Reply:
This makes it so that child extractors use the same settings as their parent by default ([`extractor.category-transfer`](https://gdl-org.github.io/docs/configuration.html#extractor-category-transfer)). Not really necessary here I think, but it probably won't hurt to have this enabled just in case.